context/app/static/js/shared-styles/snackbars/index.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import Snackbar from '@mui/material/Snackbar';
+
+import { useSnackbarStore, SnackbarProvider, createStore } from './store';
+
+const SuccessSnackbar = styled(Snackbar)`
+  & > div {
+    background-color: ${(props) => props.theme.palette.success.main};
+    color: ${(props) => (props.$isWarning ? '#000000' : props.theme.palette.white.main)};
+  }
+`;
```

Comment:
1. Let's make a `style.js` file to hold this
2. (Optional) - The `isWarning` prop on a `Success` snackbar could be a bit confusing since these are contradictory statuses. 
    * Two options would be to either split this into two separate components for the time being (SuccessSnackbar and WarningSnackbar)
    * Or, rename to `StyledSnackbar` and pass in a `severity` prop like MUI Alerts, with appropriate styles for each case ('info', 'warning', 'success', 'error') but that may be a bit more of a lift
3. (Optional) We should use `import { styled } from '@mui/styles'` instead of `styled-components` (since we're moving towards using MUI's styling functionalities), but if that doesn't work without making any additional changes feel free to skip this for now.